src/components/map/ha-entity-marker.ts

```diff
@@ -56,6 +56,9 @@ class HaEntityMarker extends LitElement {
       .marker.picture {
         overflow: hidden;
       }
+      .marker {
```

Comment:
There is already`.marker` above on line 43. Just add it there

Reply:
Sorry, was fast and hence inaccurate..